src/containers/App/App.js

```diff
@@ -5,28 +5,33 @@ import ControlForm from '../ControlForm/ControlForm';
 import ProjectContainer from '../ProjectContainer/ProjectContainer';
 import '../../main.scss';
 import { connect } from 'react-redux';
-import { fetchProjects } from '../../thunks/fetchProjects'
+import { fetchProjects } from '../../thunks/fetchProjects';
 
 export class App extends Component {
 
   componentDidMount = () => {
-    this.props.fetchProjects()
+    this.props.fetchProjects();
   };
 
   render() {
+    const { searching } = this.props;
     return (
       <div>
         <Header />
-        <Palette />
-        <ControlForm />
+        {!searching && <Palette />}
+        {!searching && <ControlForm />}
```

Comment:
Nice, I like your addition of the searching action/reducers to handle displaying these components.